tv_apps/smart-home/js/card_picker.js

```diff
@@ -285,12 +370,19 @@
 
       if (this._folder.isEmpty()) {
         this._cardManager.removeCard(this._folder);
+      } else {
+        this._folder.name.raw = this.input.value;
```

Comment:
Use `this._folder.name = {raw: this.input.value}` to remove redundant properties (if any).